docs/json.md

```diff
@@ -154,6 +154,9 @@ If a field is to be declared as "required" in the JSON Schema, the annotation
 ```
 is used.
 
+Please note, we do not recommend using the `$required` flag for boolean fields, as not setting it is equivalent to `false`. 
+If you want the field to always be visible in the JSON, you can use the annotation `$showAlways`. 
```

Comment:
Maybe, the recommendation should be more like this. If the field is crucial to be serialized (even if the value is `false`), you can use the annotation `$showAlways`. What do you think?

Reply:
Yes, sounds better as it should not get the standard to add `$showAlways` to all boolean fields.